universal-application-tool-0.0.1/test/services/applicant/ApplicantServiceImplTest.java

```diff
@@ -215,6 +216,30 @@ public void createApplicant_createsANewApplicant() {
     assertThat(applicant.id).isNotNull();
   }
 
+  @Test
+  public void createApplicant_ApplicantTime() {
+    Instant a = Instant.now();
+
+    try {
+      Thread.sleep(5);
+    } catch (InterruptedException e) {
+
+    }
+
+    Applicant applicant = subject.createApplicant(1l).toCompletableFuture().join();
+
+    try {
+      Thread.sleep(5);
+    } catch (InterruptedException e) {
+
+    }
```

Comment:
question: is it possible to avoid thread sleeping in the test? We could either try to
1. Just compare `Applicant.whenCreated` to make sure it is before `Instant.now()`
2. Create a fake `Clock`, then use it in [`now(clock)`](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#now-java.time.Clock-) to create our own version of now that we can assert on